docs/experimental_analytics_report.md

```diff
@@ -0,0 +1,27 @@
+---
+title: experimental_analytics_report
+description: "A Tiltfile builtin for better understanding your team's analytics"
+layout: docs
+---
+## Summary
+Tilt 0.17.4 adds the `experimental_analytics_report` Tiltfile builtin. This allows teams working with Tilt to better measure and improve their Tilt usage.
```

Comment:
```suggestion
[Tilt 0.17.4](https://github.com/tilt-dev/tilt/releases/tag/v0.17.4) adds the [`experimental_analytics_report` Tiltfile builtin](https://github.com/tilt-dev/tilt/pull/3751). This allows teams working with Tilt to better measure and improve their Tilt usage.
```

Reply:
I had actually put links there to start, but on second thought figured that they didn't add much information that's relevant to users.

Hm, I think the link to Tilt 0.17.4 is fine, since that's a reasonable expectation when someone clicks on Tilt 0.17.4. I don't think linking to the PR is useful. As a user, I'd expect that to link to the API docs or some other kind of user-facing doc. AFAICT, the PR doesn't add any information and clicking on it would just be a disappointment and waste of the user's time. The PR feels like a "how the sausage is made" thing that I wouldn't expect to generally be important to our audience, and is especially uninteresting in this case. Do you have an angle I'm missing, or a different set of values we should align on?

Reply:
@landism : Purely a non-blocking suggestion. Discard if you'd prefer.